.travis/cirp/check_precondition.sh

```diff
@@ -1,3 +1,19 @@
+#   Copyright © 2019 by The qTox Project Contributors
```

Comment:
I don't think we have the Copyright to these files, because they are part of @nurupo's publish tool: https://github.com/nurupo/ci-release-publisher/tree/master/scripts

Reply:
Removed all notices in these files.